google/cloud/forseti/scanner/scanners/base_scanner.py

```diff
@@ -65,13 +65,18 @@ def _output_results_to_db(self, violations):
         Returns:
             list: Violations that encountered an error during insert.
         """
+        model_description = (
+            self.service_config.model_manager.get_description(self.model_name))
+        inventory_index_id = (
```

Comment:
This can be changed to source_id = "{}-{}".format(model_description.get(source), model_description.get('source_info').get('inventory_index_id'))), if we want to have a generic id-name for scanning result.

Reply:
Please see comment below.